unitTests/ui/ComponentEventsTest.ts

```diff
@@ -27,6 +28,43 @@ export function ComponentEventsTest() {
       $$(test.env.root).trigger('foo', { bar: 'baz' });
       expect(spy).toHaveBeenCalledWith({ bar: 'baz' });
     });
+    describe('when the event is  wrapped by jQuery and does not have any param', () => {
+      beforeAll(() => {
+        window['Coveo']['$'] = $;
+      });
+
+      beforeEach(() => {
+        test.cmp.enable();
+        test.cmp.bind.on(test.env.root, 'click', spy);
+      });
+
+      it('when triggering a native click, it calls the spy with native event as the only parameter', function() {
```

Comment:
Should this test be outside of the `describe` block? It seems to contradict the description. Or is this native click wrapped inside a jQuery click?

If this test is meant to check the new lines 217-219, are you sure it is hitting those lines?

Reply:
Yep, the behavior is impacted by the before (i.e. loading of jQuery), I'll update the description of the describe to be clearer.

The lines 217 to 219 are hit and yes, the event is wrapped by jQuery.